### ISSUE TITLE
Issues with chrome and firefox

### ISSUE THREAD

**Issue by 0penwir3** [kgretzky/evilginx2#247]
Has anyone noticed any oddities with Chrome and Firefox? I have been using the Okta phishlet for internal testing with no issues across Chrome, Firefox, and IE for about a week now. For whatever reason, with no changes made, Chrome and Firefox stopped working yesterday. The page appears to load (I can inspect the site's source and see the full html as if it loaded) but the page itself is not rendering. That being said, it works fine in IE and Edge with no issues. I dropped into debug mode when launching evilginx and everything looks the same when it loads on IE as when it does not on Chrome or Firefox. On very rare occasion when Chrome is trying to load I get a TLS error but most of the time I see the expected message "New visitor has arrived ... Chrome" but the page is not rendered. I see something very similar with Modlishka. Trying to rule out an issue on my side or a global change to the browsers.

Any one else experiencing this?

**Comment by kgretzky** [comment]
This happens for all websites or just okta? I haven't seen behavior like this.

**Comment by 0penwir3** [comment]
It only appears to be Okta. 

I tested Twitter to make sure and have attached screen shots for reference. The footer of Okta is loading but nothing else. I left the domain name the same for the sake of testing. Below is Twitter in Chrome (working), Okta in IE (working), and Okta in Chrome (with the partially rendered page).

![OktaChrome](https://user-images.githubusercontent.com/42387473/54759087-5f383f00-4bbb-11e9-8f1c-a51192289952.png)

![twitterChrome](https://user-images.githubusercontent.com/42387473/54759093-62cbc600-4bbb-11e9-956d-418f729fd613.png)

![OktaIE](https://user-images.githubusercontent.com/42387473/54759565-55fba200-4bbc-11e9-9ca0-6a0fe07d9a60.png)

**Comment by 0penwir3** [comment]
I was able to get this working but it's still pretty stange. I believe Okta has done something to my phishing domain due to the sheer number of requests I made while testing. Still weird that IE is able to render but Chrome and IE cannot. I used an old domain I have for the sake of testing and everything is functioning as intended now. Not sure what the deal is... Will post any other updates in case someone else has similar issues.

![OktaChrome2](https://user-images.githubusercontent.com/42387473/54838815-7fd5c700-4c97-11e9-8c2c-5a6f4506c2d1.png)
